docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -281,63 +281,117 @@ role ``meta/argument_specs.yml`` file. All fields are lower-case.
+    - List of option values.
+    - list
+    - No
+    - []
+  * - elements
+    - Specifies the data type for list elements when type is ``list``.
+    - str
+    - No
+    - null
+  * - options
+    - If this option takes a dict or list of dicts, you can define the structure here.
+    - dict (recursively use this same table)
+    - No
+    - null
+  * - version_added
+    - For roles in collections, the version of the collection this option was added in.
+      Should only be specified if this is different from the ``version_added`` value of the entrypoint itself.
+    - str
+    - No
+    -
+  * - version_added
+    - For roles in collections, the version of the collection this option was added in.
+      Should only be specified if this is different from the ``version_added`` value of the entrypoint itself.
+    - str
+    - No
+    -
 
-        * A short, one-line description of the entry point.
+.. note::
         * The ``short_description`` is displayed by ``ansible-doc -t role -l``.
-
-    :description:
-
-        * A longer description that may contain multiple lines.
-
-    :author:
-
-        * Name of the entry point authors.
-        * Use a multi-line list if there is more than one author.
-
-    :options:
-
-        * Options are often called "parameters" or "arguments". This section defines those options.
-        * For each role option (argument), you may include:
-
-        :option-name:
-
-           * The name of the option/argument.
-
-        :description:
-
-           * Detailed explanation of what this option does. It should be written in full sentences.
-
-        :type:
-
-           * The data type of the option. See :ref:`Argument spec <argument_spec>` for allowed values for ``type``. Default is ``str``.
-           * If an option is of type ``list``, ``elements`` should be specified.
-
-        :required:
-
-           * Only needed if ``true``.
-           * If missing, the option is not required.
-
-        :default:
-
-           * If ``required`` is false/missing, ``default`` may be specified (assumed 'null' if missing).
-           * Ensure that the default value in the docs matches the default value in the code. The actual
-             default for the role variable will always come from ``defaults/main.yml``.
-           * The default field must not be listed as part of the description, unless it requires additional information or conditions.
-           * If the option is a boolean value, you should use `true/false` if you want to be compatible with `ansible-lint`.
-
-        :choices:
-
-           * List of option values.
-           * Should be absent if empty.
-
-        :elements:
-
-           * Specifies the data type for list elements when type is ``list``.
-
-        :options:
-
-           * If this option takes a dict or list of dicts, you can define the structure here.
+        * description is not required by Ansible itself but antsibull-docs does expect it to be there to build documentation.
```

Comment:
I don't know if this is a reasonable requirement to put people.
At this time there are multiple instances where ansible code itself isn't compliant[0] 

[0]https://github.com/search?q=description%3A++path%3A**%2Fargument_specs.yml+owner%3Aansible&type=code&ref=advsearch

Reply:
Also, if at all possible, the type should allow for a description consisting of a string. Multiline strings beat lists in both readability and ease of writing.

Reply:
A multiline string is still a subset of a list of strings (just a list with one element)

Reply:
No it isn't compliant, because many of those file don't have `description` key at all. 
For example:
https://github.com/ansible/instruqt/blob/a510499256cdafcb23e8a38c47ffd7343396ab21/images/ansible/roles/vs_code/meta/argument_specs.yml#L4